content/apollo.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Apollo
+order: 15
+description: The Apollo data stack for Reactive GraphQL
+discourseTopicId: TODO
+---
+
+<h2 id="introduction">Introduction</h2>
+
+Apollo is Meteor's next-generation data stack, built with GraphQL. While it doesn't yet have all the features that Meteor's pub/sub system has, it provides a way to get data from any database – not just MongoDB.
```

Comment:
> Apollo is Meteor's next-generation data stack 

I don't believe this open statement is correct.

Apollo is not part of Meteor or a next generation of Meteor